### PR TITLE
elements: fingerprints, login, routing, statistics. scripts: util. de…

### PR DESCRIPTION
**Pull request by victorbalan**
elements: fingerprints, login, routing, statistics. scripts: util. dep: google-chart

Added a basic login on /login wich sends a mocked login request to get the sessionid to authorize requests

### CODE REVIEW

File: app/elements/fingerprints/fingerprints.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        }
+      },
+      onResponse: function(response){
+        this.items = response.detail.response
+      },
+      onError: function(error){
+        console.log(error.detail)
+      },
+      ready: function(){
+        this.requestUrl = "http://localhost:8080/api/fingerprint/company/" + getCookie("companyId");
```
Comment (flowlo) [inline app/elements/fingerprints/fingerprints.html]: Shouldn't this just use something like `window.location` to build the endpoint path?!

File: app/elements/login/login.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      properties:{
+        invalidLogin:{
+          type: Boolean,
+          notify: true
+        }
+      },
+      login: function(){
+        this.$.ajaxRequest.method='POST';
+        this.$.ajaxRequest.body = JSON.stringify({'company': "Catalysts"});
+        this.$.ajaxRequest.url='http://localhost:8080/api/company/login';
```
Comment (flowlo) [inline app/elements/login/login.html]: Hardcoded localhost.

File: app/elements/login/login.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      is: 'login-page',
+      properties:{
+        invalidLogin:{
+          type: Boolean,
+          notify: true
+        }
+      },
+      login: function(){
+        this.$.ajaxRequest.method='POST';
+        this.$.ajaxRequest.body = JSON.stringify({'company': "Catalysts"});
```
Comment (flowlo) [inline app/elements/login/login.html]: Hardcoded company name.

Comment (victorbalan) [inline app/elements/login/login.html]: yes..this is for the basic login untill we figure out what we need for login.

Comment (flowlo) [inline app/elements/login/login.html]: E-Mail address and password. To be checked in the backend against known costumers.

File: app/elements/fingerprints/fingerprints.html (hunk 1 above)
Comment (victorbalan) [inline app/elements/fingerprints/fingerprints.html]: why??

**Pull request merged**